src/app/EventManagement.cpp

```diff
@@ -359,6 +360,10 @@ CHIP_ERROR EventManagement::ConstructEvent(EventLoadOutContext * apContext, Even
 
     // Callback to write the EventData
     ReturnErrorOnFailure(apDelegate->WriteEvent(apContext->mWriter));
+    if (!(apOptions->mFabricIndex.IsNull()))
```

Comment:
OK, so this is definitely an `Optional`, not a `Nullable`....

Reply:
Currently these LogEvent template function are unifying using nullable fabricIndex, preferring still to use nullable fabricIndex here, thanks

Reply:
update with nullable